packages/downloads/src/utils.ts

```diff
@@ -49,3 +53,17 @@ export function composeDownloadId(params: IDownloadIdParams): string {
   const { datasetId, format, spatialRefId, geometry, where } = params;
   return `${datasetId}:${format}:${spatialRefId}:${geometry}:${where}`;
 }
+
+/**
+ * @private
+ */
+export function isRecentlyUpdated(
+  target: DownloadTarget,
+  lastEditDate: number
+): boolean {
+  return (
+    target === "portal" &&
```

Comment:
If this util is only ever used for portal downloads, maybe it should be in the `/portal` directory?

Reply:
I can go with that. This is now in a `utils` file in `/portal`